constants.go

```diff
@@ -522,6 +522,10 @@ const (
 	// allowed database users.
 	TraitDBUsers = "db_users"
 
+	// TraitTeams is the name of the role variable use to store team
+	// membership information
+	TraitTeams = "teams"
```

Comment:
Are we ok with just `teams` here rather than `github_teams`?

Reply:
I think so -- in the issue this was the name that was suggested by @russjones